.ci/magician/cmd/generate_comment.go

```diff
@@ -296,34 +296,25 @@ func execGenerateComment(prNumber int, ghTokenMagicModules, buildId, buildStep,
 	}
 
 	// Run missing test detector (currently only for beta)
-	missingTestsPath := mmLocalPath
 	for _, repo := range []source.Repo{tpgbRepo} {
 		if !repo.Cloned {
 			fmt.Println("Skipping missing tests; repo failed to clone: ", repo.Name)
 			continue
 		}
-		missingTests, err := detectMissingTests(missingTestsPath, repo.Path, oldBranch, rnr)
+		err = buildDiffProcessor(diffProcessorPath, repo.Path, diffProcessorEnv, rnr)
```

Comment:
rather than rebuilding the diff processor, could you use the diff processor that's built above? (For now, by explicitly checking if the repo is beta before running missing test detection. Alternatively, we could run it for both and just discard one of them if both give responses, since we're not changing the formatting of the response as part of this.